.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Maybe hint that error log and browser console are very helpful too - with a link to https://metabase.com/docs/latest/troubleshooting-guide/bugs.html (which needs an update)